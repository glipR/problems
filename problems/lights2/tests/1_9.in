160090761011141772