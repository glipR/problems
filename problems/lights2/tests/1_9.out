106727174007427847
